docs/agents/smurf_crate_monitor.rst

```diff
@@ -0,0 +1,87 @@
+.. highlight:: rst
+
+.. _smurf_crate_monitor:
+
+=========================
+Smurf Crate Monitor Agent
+=========================
+
+The SMuRF readout system uses Advanced Telecommunications Computing Architecture 
+(ATCA) crates for powering and communicating between boards and the site networking
+and timing infrastructure. These crates have a small computer on board called a 
+shelf manager which monitors all of the sensors in the crate including ammeters, and
+voltmeters for the power into the crates and into each front and rear module of each
+active slot used in the crate. There are also tachometers on each of the crate fans
+and various thermometers withing the crate and each of the boards plugged into the
+crate which the shelf manager monitors. There are multiple crate manufacturers
+but the shelf managers all share the same set of programming/communication called
+Pigeon Poing Communication so this agent should work across multiple crate
+manufacturers. This agent connects to a shell terminal of a crate shelf 
+manager over ssh through the python subprocess package and then runs the 
+command 'clia sensordata' and parses its output to identify all of the available 
+sensors then stream and publish them.
+
+Configuration File Examples
+---------------------------
+Below are configuration examples for the ocs config file and for running the
+Agent in a docker container.
+
+ocs-config
+``````````
+To configure the SMuRF Crate Monitor Agent we need to add a CrateAgent entry 
+to our site configuration file. Here is an example configuration block using 
+all of the available arguments::
+
+        {'agent-class': 'CrateAgent',
+         'instance-id': 'smurf-crate-monitor',
```

Comment:
I think we should put the crate-id in the instance-id in this example. Something like `crate1-monitor`.